File: app/src/main/java/ivankuo/com/itbon2018/di/FragmentBuildersModule.java
package ivankuo.com.itbon2018.di;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import ivankuo.com.itbon2018.ui.RepoFragment;

@Module
public abstract class FragmentBuildersModule {
    @ContributesAndroidInjector
    abstract RepoFragment contributeRepoFragment();
}

File: app/src/main/java/ivankuo/com/itbon2018/di/FragmentBuildersModule.kt
package ivankuo.com.itbon2018.di

import dagger.Module
import dagger.android.ContributesAndroidInjector
import ivankuo.com.itbon2018.ui.RepoFragment

@Module
abstract class FragmentBuildersModule {
    @ContributesAndroidInjector
    abstract fun contributeRepoFragment(): RepoFragment
}